### ISSUE TITLE
Is value required?

### ISSUE THREAD

**Issue by paltman** [davidroyer/vue2-editor#121]
I'm trying to use the editor and am following the docs so that I have:

```vue
<vue-editor v-model="content"></vue-editor>
```
in my `<template>` and `content` that is loaded into editor just fine on load.

However, I notice this warning in my console followed by some errors:

```Invalid prop: type check failed for prop "value". Expected String, got Object.```

This is technically a `Vue Warn` but message says Error.

```Error in mounted hook: "TypeError: (this.value || "").replace is not a function"``` 

Then next in console is the `TypeError`:

```
TypeError: (this.value || "").replace is not a function
     at VueComponent.filteredInitialContent (VM34543 site.js:106690)
```

Then another `TypeError`:

```
VM34543 site.js:106690 Uncaught TypeError: Cannot read property 'emit' of undefined
    at e.value (VM34543 site.js:106690)
```

I'm running:

* Vue - 2.5.16
* Vue2 Editor - 2.5.0

Thanks!

**Comment by paltman** [comment]
I think the issue is centered on that I'm using a computed getter/setting for my `v-model`:

```js
computed: {
        content: {
            get () {
                if (this._content) {
                    return this._content;
                }
                return this.$store.getters.content;
            },
            set (value) {
                this._content = value;
            }
        }
    },
```

**Comment by paltman** [comment]
Ok, I've eliminated all but the last error by wrapping the component and setting an `initialContent` property with my data from the store and adding a watcher on that property to update the local state `content` which has the `v-model` bound to it.  Now everything loads up without warning but am still getting: 

```
vue2-editor.js:1 Uncaught TypeError: Cannot read property 'emit' of undefined
    at e.value (vue2-editor.js:1)
    at MutationObserver.<anonymous> (vue2-editor.js:1)
```

**Comment by paltman** [comment]
Ok, solved.  I went back to the computed getter/setter but got rid of the warnings by checking the value of the getter before returning to make sure it was a string.  Right before it got content, it was an `Observer` object.

```js
        content: {
            get () {
                if (this._content) {
                    return this._content;
                }
                return this.$store.getters.content.length > 0 ? this.$store.getters.content : '';
            },
            set (value) {
                this._content = value;
            }
        }
```

The `emit` but was because of some gnarly HTML that had somehow gotten in the content during testing.  starting fresh has eliminated that issue.

**Comment by romines** [comment]
@paltman Should this be fixed in 2.6.6? I am seeing it in 2.6.6.

If so I will try to give you a clean reproduction.

Thanks for an excellent component.
